shell/artifacts/hex/hex-game.js

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2018 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+defineParticle(({DomParticle}) => {
+  const rootTemplate = `
+    <style>
+      hex-game {
+        --hex-p1: #F44336;
+        --hex-p2: #2196F3;
+      }
+      hex-game[player=p1] {
+        --current-player-color: var(--hex-p1);
+      }
+      hex-game[player=p2] {
+        --current-player-color: var(--hex-p2);
+      }
+
+      /* TODO: Remove this once we switch slots to use shadow dom */
+      hexa-gon > div {
+        width: 100%;
+        height: 100%;
+      }
+      hex-cell {
+        width: 100%;
+        height: 100%;
+        display: block;
+      }
+      hex-cell:hover {
+        background: var(--current-player-color);
+        opacity: 0.5;
+      }
+      hex-cell[player=p1] {
+        background: var(--hex-p1);
+        opacity: 1;
+      }
+      hex-cell[player=p2] {
+        background: var(--hex-p2);
+        opacity: 1;
+      }
+    </style>
+    hex-game <hex-game player$="{{player}}" slotid="board"></hex-game>`;
+  const cellTemplate = `<hex-cell player$="{{player}}" on-click="onCellClick"></hex-cell>`;
```

Comment:
I believe here you could have `key={{subId}}`

Reply:
Done.